integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -527,6 +525,41 @@ public void testFacetCountsReflectUserPermissions() throws Exception {
         }
     }
 
+    @Test
+    public void testSearchAllExpectedFieldsForRecordReturned() throws Exception {
+        createWorkAndFileObjects();
+
+        var getMethod = new HttpGet(SEARCH_URL + "/?types=Work,File&rollup=false");
+
+        try (var resp = httpClient.execute(getMethod)) {
+            var metadata = getMetadataFromResponse(resp);
+            assertSuccessfulResponse(resp);
+
+            assertStringValuePresent(metadata, 1, "thumbnail_url");
+            assertStringValuePresent(metadata, 1, "id");
+            assertStringValuePresent(metadata, 1, "title");
```

Comment:
It would help with readability to group the assertions on the same object together, so it's clear what the expectations are for the work versus the file.

In addition to verifying that a field is present, I think we should check that some of the fields with known, verifiable values, are populated. This could help us avoid problems if any expectations change, like empty fields return an empty string instead of nulls, etc, or we get back an array with unexpected values. I'd suggest doing this for title, resourceType, creator, format, fileType, identifier, rollup.

Reply:
I checked the values for all the fields you suggested except rollup. Rollup changed every time I ran the tests.

Reply:
The value for rollup is the id of the work, so you should be able to get it from the work object